[
    {
        "id": 219936583,
        "name": "CVE-2019-5010",
        "full_name": "JonathanWilbur\/CVE-2019-5010",
        "owner": {
            "login": "JonathanWilbur",
            "id": 20342114,
            "avatar_url": "https:\/\/avatars.githubusercontent.com\/u\/20342114?v=4",
            "html_url": "https:\/\/github.com\/JonathanWilbur"
        },
        "html_url": "https:\/\/github.com\/JonathanWilbur\/CVE-2019-5010",
        "description": "CVE-2019-5010 Exploit PoC - Python Denial of Service via Malformed X.509v3 Extension",
        "fork": false,
        "created_at": "2019-11-06T07:26:43Z",
        "updated_at": "2019-11-06T07:27:36Z",
        "pushed_at": "2022-09-08T10:56:45Z",
        "stargazers_count": 0,
        "watchers_count": 0,
        "forks_count": 0,
        "allow_forking": true,
        "is_template": false,
        "web_commit_signoff_required": false,
        "topics": [
            "denial-of-service",
            "dos",
            "exploit",
            "python",
            "x509",
            "x509certificates"
        ],
        "visibility": "public",
        "forks": 0,
        "watchers": 0,
        "score": 0
    }
]